packages/perseus/src/renderer.tsx

```diff
@@ -550,7 +550,10 @@ class Renderer extends React.Component<Props, State> {
             // filtered out in this.render(), so we shouldn't have to
             // worry about using this widget key and ref:
             return (
-                <ErrorBoundary key={"container:" + id}>
+                <ErrorBoundary
+                    key={"container:" + id}
+                    metadata={{widget_type: type}}
```

Comment:
Looking a little more at this, I think it'd be useful to add the widget id also here (the `id` variable in scope).

```suggestion
                    metadata={{widget_type: type, widget_id: id}}
```